src/SprykerEco/Zed/Billpay/Business/Payment/Request/QuoteTransactionInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Billpay\Business\Payment\Request;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+
+interface QuoteTransactionInterface extends TransactionInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return mixed
```

Comment:
Looks like this should be

    \Generated\Shared\Transfer\BillpayPrescoringTransactionResponseTransfer $billpayResponseTransfer